public/documentation/apis/presentation_api_tutorial.md

```diff
@@ -0,0 +1,195 @@
+Presentation API Tutorial
+=========================
+
+Overview
+--------
+
+[Presentation API](http://webscreens.github.io/presentation-api/) is currently a [W3C spec](http://w3c.github.io/presentation-api/) draft for secondary display support for the Web. It allows a web app to present web content on a secondary display connected to the device. It has several uses: presenting slides, sharing video and image, and playing game on a large screen TV via the mobile device.
```

Comment:
The last sentence needs a little tweaking:    "It has several uses:  presenting slides, sharing video and image(s?), and playing game(s?), ..."